tests/app/clients/test_performance_platform.py

```diff
@@ -58,3 +58,14 @@ def test_should_raise_for_status(perf_client):
     with pytest.raises(requests.HTTPError), requests_mock.Mocker() as request_mock:
         request_mock.post('https://performance-platform-url/foo', json={}, status_code=403)
         perf_client.send_stats_to_performance_platform({'dataType': 'foo'})
+
+
+def generate_payload_id(payload, param):
+    pass
```

Comment:
What is this used for?  I may be missing how it's wired up if it's meant to mock something.

Reply:
ha, it's an oops.  Early effortst to write something and then abandoned it because I ran into some issue.  Will fix.